plans/init.pp

```diff
@@ -0,0 +1,3 @@
+plan bolt_test(){
+  return bolt_test::fake_task(10,10, Target.new('fake'))
```

Comment:
TODO: Cas will help Austin and Rob create a new target per result instead of sharing one.